### PR TITLE
Discord webhook integration (#13)

### PR DESCRIPTION
**Pull request by JanKrb**
Is your feature request related to a problem? Please describe.

A group of users planning to create an official server and want a wiki log channel for post updates.

Describe the solution you'd like

Implement a discord webhook. One fixed in the .env or a table with different scopes.

Issue #13

### CODE REVIEW

File: app/Channels/DiscordMessage.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+     *
+     * @param  string $content The content.
+     * @return \App\Channels\DiscordMessage
+     *
+     * @throws \InvalidArgumentException Thrown on empty content.
+     */
+    public function content(string $content): self
+    {
+        if (!strlen($content)) {
+            throw new \InvalidArgumentException('Content must not be empty.');
```
Comment (NinoKrb) [inline app/Channels/DiscordMessage.php:64]: Please translate the Exception message

File: app/Notifications/Webhook/Discord/PostUpdated.php [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param  mixed  $notifiable
+     */
+    public function toDiscord($notifiable)
+    {
+        return DiscordMessage::create()
+            ->username(config('app.name', 'Articly'))
```
Comment (NinoKrb) [inline app/Notifications/Webhook/Discord/PostUpdated.php:42]: Maybe make usernames customizable in a config

File: app/Notifications/Webhook/Discord/PostUpdated.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            ->embeds([
+                [
+                    "title" => 'Post Changed: ' . $this->post->title,
```
Comment (NinoKrb) [inline app/Notifications/Webhook/Discord/PostUpdated.php:45]: Title message maybe make customizable in a config

**Review by NinoKrb — CHANGES_REQUESTED** [review]
Please check again about possible translations and customizable config parameters
Title message maybe make customizable in a config.

File: app/Notifications/Webhook/Discord/PostUpdated.php (hunk 2 above)
Comment (JanKrb) [reply app/Notifications/Webhook/Discord/PostUpdated.php:42]: Notification will take config's `app.name` or env's `APP_NAME` and `Articly` as a default.

File: app/Channels/DiscordMessage.php (hunk 1 above)
Comment (JanKrb) [reply app/Channels/DiscordMessage.php:64]: Exceptions are thrown and shown in the log. Translation only matters for user output.

File: app/Notifications/Webhook/Discord/PostUpdated.php (hunk 3 above)
Comment (JanKrb) [reply app/Notifications/Webhook/Discord/PostUpdated.php]: Fixed in https://github.com/KrbDevelopment/Articly-Backend/pull/14/commits/907e9af8d1becc97aaf0a85033b469d78c071f18

**Review by NinoKrb — APPROVED** [review]

**Pull request merged**